tools/advisor/advisor/db_benchmark_client.py

```diff
@@ -69,40 +36,82 @@ def __init__(self, client_binary_path, environment_variables):
             self.environment_variables = environment_variables
         self.get_available_workloads()
 
-    def _run_command(self, command, cleanup=True):
-        print('executing...')
-        print(command)
+    def _get_log_options(self, db_options):
+        # get the location of the LOG file and the frequency at which stats are
+        # dumped in the LOG file
+        log_dir_path = None
+        stats_freq_sec = None
+        # options to fetch
+        dump_period = 'DBOptions.stats_dump_period_sec'
```

Comment:
How do you set it this before running dbbench experiments?

Reply:
I set it in the DatabaseOptions object that I use when I call run_experiment(options) (reference: config_optimizer_example.py)